File: PROD/Exercise6/Kamila.py
#import time
#t1 = time.time()
def Sum_square_diff(x): 
    y = x + 1
    z = 2*x + 1 
    sumofsquares = (x*y*z)/6
    squareofsum = ((x*y)/2)**2
    return int(squareofsum-sumofsquares)

print (Sum_square_diff(100))
#t2=time.time()
#print(t2-t1)

File: PROD/Exercise6/Kamila.cs
using System; 

namespace Exercise6
{
    class Kamila : Exercise
    {
        public int Run()
        {
            double x = 100;
            double y = x + 1;
            double z = 2*x + 1; 
            double sumofsquares = (x*y*z)/6;
            double squareofsum = Math.Pow((x*y)/2,2);
            return (int)(squareofsum-sumofsquares);
        }
    }
}